server/routes/polls/polls.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+const authentication = require('../../middleware/authentication');
+const PollsMongoModel = require('../../models/Polls');
+const { cloudinary } = require('../../cloudinary/cloudinary');
+
+router.post('/create', authentication, async (req, res) => {
```

Comment:
For consistency with the other backend routes, maybe we can make this a POST request to /polls
```suggestion
router.post('/polls', authentication, async (req, res) => {
```